__tests__/e2e/package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "e2e-synthetics",
+  "private": true,
+  "main": "index.js",
+  "license": "MIT",
+  "scripts": {
+    "setup": "sh scripts/setup.sh",
```

Comment:
This should either be `scripts/setup.sh` or `bash scripts/setup.sh`. That shell script requires bash features not present in the minimal `sh` binary.